Booting/linux-bootstrap-2.md

```diff
@@ -446,17 +465,19 @@ where `0x80` is the first hard drive and the `EDD_MBR_SIG_MAX` macro is 16. It c
 Conclusion
 --------------------------------------------------------------------------------
 
-This is the end of the second part about linux kernel internals. In the next part we will see video mode setting and the rest of preparations before transition to protected mode and directly transitioning into it.
+This is the end of the second part about Linux kernel internals. In the next part we will see video mode setting and the rest of preparations before transition to protected mode and directly transitioning into it.
 
 If you have any questions or suggestions write me a comment or ping me at [twitter](https://twitter.com/0xAX).
 
 **Please note that English is not my first language, And I am really sorry for any inconvenience. If you found any mistakes please send me a PR to [linux-internals](https://github.com/0xAX/linux-internals).**
 
-Links
+Links For Further Reading
```

Comment:
Can you please remove `For Further Reading`, they are not for further reading, but just can help to understand all material from the post